sig-cluster-lifecycle/annual-report-2022.md

```diff
@@ -4,64 +4,93 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
-
+   - We migrated the default image repository in kubeadm and some of our other k8s deployment
+   tools from k8s.gcr.io to registry.k8s.io.
+   - kubeadm initiated work on migrating its etcd bootstrapping to use ["learner mode"](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/kubeadm/3614-etcd-learner-mode)
+   - We finilized migrating subprojects away from the legacy ["master" label/taint](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/kubeadm/2067-rename-master-label-taint)
```

Comment:
there are two kubeadm KEP updates in 2022 that are not mentioned here. 
- [KEP-2915: Replace usage of the kubelet-config-x.y naming in kubeadm](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/kubeadm/2915-kubeadm-replace-kubelet-config-x.y), using UnversionedKubeletConfigMap alpha:v1.23; beta: v1.24; GA: v1.25; Removed: v1.26
- [KEP-1739: kubeadm customization with patches](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/kubeadm/1739-customization-with-patches): [kubeadm: add support for patching a "kubeletconfiguration" target #110405](https://github.com/kubernetes/kubernetes/pull/110405/files) v1.25